.github/workflows/build.sh

```diff
@@ -37,6 +37,10 @@ else
 fi
 
 mdbook build -d "$dest_dir"
+
+# Disable the redbox button in built versions of the course
+echo '// disabled' > "${dest_dir}/html/theme/redbox.js"
```

Comment:
Thanks, this is a great idea! I would refer back to the script like this:

```suggestion
echo '// Disabled in published builds, see build.sh' > "${dest_dir}/html/theme/redbox.js"
```